### PR TITLE
Update travis

### PR DESCRIPTION
**Pull request by csgillespie**
Update to use bionic and also form template for other packages
 - Use alerts+travis@ for email to allow better filtering
 - Use RSPM, but need to install Rcpp due to bug

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,44 @@
...
     on_success: change
     on_failure: change
 
-before_install:
-  - sudo apt-get install libgit2-dev
+# Generic + Bespoke
+# 1. Only needed for jrNotes
+# 2. Rcpp bugs. On next update of Rcpp CRAN, delete line 2
+before_install: |
+  sudo apt-get install libgit2-dev
```
Comment (csgillespie) [inline .travis.yml:23]: For jrNotes only

File: .travis.yml [hunk 2]
```diff
@@ -1,29 +1,44 @@
...
+  Rscript -e 'if (!requireNamespace("Rcpp", quietly = TRUE)) install.packages("Rcpp")'
```
Comment (csgillespie) [inline .travis.yml:24]: Rcpp bug. Remove in next Rcpp update on CRAN

File: .travis.yml [hunk 3]
```diff
@@ -1,29 +1,44 @@
...
   global:
     - ALLOWED_NOTES=1
     - NO_IMPORTS=0
 
+# Generic
 notifications:
   email:
     recipients:
-     - alerts@jumpingrivers.support
+     - alerts+travis@jumpingrivers.support
```
Comment (csgillespie) [inline .travis.yml:15]: Allow better email filtering

**Comment by JohnMcIntyreJR** [comment]
@csgillespie Can you remind me what the importance of updating to bionic is?

**Comment by csgillespie** [comment]
> @csgillespie Can you remind me what the importance of updating to bionic is?
Bionic is Ubuntu 20.04. 

Currently travis is on 16.04, which is over four years out of date.

**Review by JohnMcIntyreJR — APPROVED** [review]

**Comment by JohnMcIntyreJR** [comment]
@csgillespie Approved, so ready for merging if no further changes are required

**Review by trianglegirl — APPROVED** [review]

**Pull request merged**